engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -752,4 +753,8 @@ ValidationException getContainerElementTypeHasAlreadyBeenConfiguredViaXmlMapping
 
 	@Message(id = 218, value = "Having parallel definitions of ValueExtractor on a given class is not allowed: %s.")
 	ValueExtractorDefinitionException getParallelDefinitionsOfValueExtractorException(@FormatWith(ClassObjectFormatter.class) Class<?> extractorImplementationType);
+
+	@Message(id = 219, value = "Unable to get the most specific value extractor for type %1$s as several value extractors are defined in this type's parallel hierarchies: %2$s.")
+	ConstraintDeclarationException unableToGetMostSpecificValueExtractorDueToSeveralValueExtractorsDefinedForParallelHierarchies(@FormatWith(ClassObjectFormatter.class) Class<?> valueType,
```

Comment:
Should we rather show the list of extractor classes (as that's more familiar to the user than our internal descriptor type).